src/app/_components/Feedback.js

```diff
@@ -54,7 +54,7 @@ export function Feedback({ isFeedbackOpen, onClose }) {
   };
 
   useEffect(() => {
-    if (isFormOpen) return null;
+    if (isFormOpen) return undefined;
```

Comment:
You do not need to explicitly return undefined, just do `return;`

Reply:
You're absolutely right!) fixed it